installing/getting-started-rhacs-cloud-ocp.adoc

```diff
@@ -119,4 +119,38 @@ If no data appears in the ACS Console:
 * Ensure that at least one secured cluster is connected to your {product-title-managed-short} instance. For more information, see the "Installing secured cluster resources on each cluster" section.
 * Examine your Sensor pod logs to ensure that the connection to your {product-title-managed-short} instance is successful.
 * In the {ocp-short} cluster, navigate to *Platform Configuration* -> *Clusters* to verify that the components are healthy and view additional operational information.
-* Examine the values in the `SecuredCluster` API in the Operator on your local cluster to ensure that the *Central API Endpoint* has been entered correctly. This value should be the same value as shown in the *ACS instance* details in the {cloud-console}.
\ No newline at end of file
+* Examine the values in the `SecuredCluster` API in the Operator on your local cluster to ensure that the *Central API Endpoint* has been entered correctly. This value should be the same value as shown in the *ACS instance* details in the {cloud-console}.
+
+[id="default-access-acs-console"]
+== Default access to ACS Console
+
+By default, authentication mechanism available to users is authentication by using Red Hat Single Sign-On (SSO).
+You cannot delete or change the Red Hat SSO authentication provider.
+However, you can change the minimum access role and add additional rules, or add another Identity provider.
+
+For each RHACS web console, dedicated OIDC client of `sso.redhat.com` is created. All OIDC clients share the same `sso.redhat.com` realm.
+We map next claims from the token issued by `sso.redhat.com` to ACS-issued token:
+
+* `realm_access.roles` to `groups`
+* `org_id` to `rh_org_id`
+* `is_org_admin` to `rh_is_org_admin`
+* `sub` to `userid`
+
+The built-in Red Hat SSO authentication provider has the required attribute `rh_org_id` set to the organization ID assigned to account of the user who created the ACSCS instance.
+This is the ID of the organizational account the user is a part of. This can be thought of as the "tenant" the user is under and owned by.
+Only users with the same organizational account can access the {product-title-short} console by using the Red Hat SSO authentication provider.
+
+[NOTE]
+====
+To gain more control over access to your {product-title-short} web console, it is highly recommended not to rely on Red Hat SSO auth provider.
+You can set up your own Identity provider right after provisioning using one of the guides under `Operating -> Managing user access`. *Pro tip*: for this authentication provider to be the first authentication option on the login page, its name should be lexicographically smaller than `Red Hat SSO`.
+====
+
+The minimum access role is set to `None`. Assigning a different value to this field will give access to the {product-title-short} instance to all users with the same organizational account.
```

Comment:
```suggestion
The minimum access role is set to `None`. Assigning a different value to this field gives access to the ACSCS instance to all users with the same organizational account.
```

Reply:
For now, I'm leaving the product names in, but we have variables for them.

Reply:
@kcarmichael08 do you want to reuse product names within this PR?

Reply:
We can if you don't mind? They are:

```
:rh-rhacscs-first: Red Hat Advanced Cluster Security Cloud Service (RHACS Cloud service)
:product-title-managed-short: RHACS Cloud Service
```

Reply:
Although I am not sure the second one is correct, but I guess the idea is we can change the variable later if the product name gets tweaked.